internal/testplanet/node.go

```diff
@@ -73,9 +75,22 @@ func (node *Node) DialPointerDB(destination *Node, apikey string) (pdbclient.Cli
 		}
 		return piececlient.NewPSClient
 	*/
+
+	// TODO: handle disconnect
 	return pdbclient.NewClient(node.Identity, destination.Addr(), apikey)
 }
 
+// DialOverlay dials destination with apikey and returns pointerdb Client
```

Comment:
This comment seems to be copied from `DialPointerDB` and is not updated accurately.

Reply:
fixed